src/pages/components/modal/usage.mdx

```diff
@@ -1,103 +1,256 @@
+</Column>
+</Row>
 
-#### Footer
+#### Title as message
 
-The footer area of a modal typically contains either one or two [buttons](/components/button/code). Do not include three buttons in the footer of your modal. If you need to include a “help” or other non-primary action, include it as a link in the modal's body.
+For short, direct messages the title can include the whole message to add visual clairty to an otherwise repeativite title and body message. When using this style,no other body copy may be included.
 
 <Row>
 <Column colLg={8}>
 
-![Example of modal with one button.](images/modal-usage-4a.png)
+![Modal sizes](images/modal-usage-title.png)
 
 </Column>
 </Row>
 
+### Max heights
+
+Each modal size has a max height in order to maintain a proper window ratio. If your modal has too much scrolling due to a max height consider using the next modal size up. If the large modal height is still not enough space then this is an indicator that a full page may be needed instead.
+
+<!--
+| Modal size | Max height |
+| ---------- | ---------- |
+| XS         | 48% |
+| Small | 72% |
+| Default | 84% |
+| Large | 96% |
+-->
+
 <Row>
 <Column colLg={8}>
 
-![Example of modal with two buttons.](images/modal-usage-4b.png)
+![Modal sizes](images/modal-usage-max-heights.png)
 
 </Column>
 </Row>
 
+#### Scrolling behavior
+
+When the modal content is longer than the modal height then the body section should scroll vertically with the header and footer remaining fixed in place. The content should visible fade at the end of the modal body area to indicate there is additional content out of view. Modal content should never horizontal scroll, instead use a larger size modal.
+
 ## Variations
 
+There are four main variations of modal.
+
+| Variant                                                        | Usage                                                                                                                           |
+| -------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
+| [Passive](/components/modal/usage#passive-modal)               | Presents information the user needs to be aware of concerning their current workflow. Contains no actions for the user to take. |
+| [Transactional](/components/modal/usage#transactional-modal)   | Requires an action to be taken in order for the modal to be completed and closed. Contains a cancel and primary action buttons. |
+| [Acknowledgment](/components/modal/usage#acknowledgment-modal) | System requires an acknowledgement of the information from the user. Contains only a single button, commonly "OK."              |
+| [Progress](/components/modal/usage#progress-modal)             | Requires several steps to be completed before it can be closed. Contains a cancel, previous and next/completion buttons.        |
```

Comment:
```suggestion
| [Progress](/components/modal/usage#progress-modal)             | Requires several steps to be completed before it can be closed. Contains a cancel, previous, and next/completion buttons.        |
```